jobio-back/utils/functions.js

```diff
@@ -0,0 +1,12 @@
+
+function formatDate(date) {
```

Comment:
i would recommend using a package for handling and formatting dates. something like date-fns is really good.